.github/workflows/qa.yml

```diff
@@ -26,19 +26,23 @@ jobs:
         include:
           - version: 7/community
             tag: 7-community
+            test: docker
           - version: 8/community
             tag: 8-community
+            test: docker
           - version: 8/developer
             tag: 8-developer
+            test: docker
           - version: 8/enterprise
             tag: 8-enterprise
+            test: docker
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Build image
         run: docker build -t "sonarqube:${{ matrix.tag }}" "${{ matrix.version }}"
       - name: Test image
-        run: ./run-tests.sh "sonarqube:${{ matrix.tag }}"
+        run: ./run-tests.sh "sonarqube:${{ matrix.tag }}" "${{ matrix.test }}"
```

Comment:
`matrix.test` is always `"docker"`

I would recommend to use the string directly.